twoliter/embedded/Makefile.toml

```diff
@@ -186,6 +186,15 @@ BUILDSYS_OVF_TEMPLATE = "${BUILDSYS_ROOT_DIR}/variants/${BUILDSYS_VARIANT}/templ
 # The default name of uploaded OVAs; override by setting VMWARE_VM_NAME
 VMWARE_VM_NAME_DEFAULT = "${BUILDSYS_NAME}-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${BUILDSYS_VERSION_IMAGE}-${BUILDSYS_VERSION_BUILD}"
 
+# Depends on ${BUILDSYS_ARCH} and ${BUILDSYS_VARIANT}.
+FETCH_VARIANT_LOCAL_ROOT_JSON = { script = ['echo "${FETCH_VARIANT_LOCAL_ROOT_JSON:-""}"'] }
+FETCH_VARIANT_ROOT_JSON = { script = ['echo "${FETCH_VARIANT_ROOT_JSON:-https://cache.bottlerocket.aws/root.json}"'] }
+FETCH_VARIANT_ROOT_JSON_SHA512 = { script = ['echo "${FETCH_VARIANT_ROOT_JSON_SHA512:-a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0a4698bd54c2f125493f4669610a9da83a1787eb58a8303b2ee488fa2a3f7d802f}"'] }
```

Comment:
We could also specify a specific root.json, like `root-4.json`, which would keep matching the sha and would allow us to pull future root.jsons in the chain.

I agree this is a foot-gun as is.

Reply:
I think we just have to be diligent in updating. Even if there was a lapse between updates, the work around would be to pass `-e  FETCH_VARIANT_ROOT_JSON_SHA512=<NEW_SHA>`. An alternative is we just have the hash somewhere in the bottlerocket repo and then this logic and any quickstart guides we have can just point to that hashes file. We can even keep track of older roots in said file.